docs/_articles/en/user-guide/prettier.md

```diff
@@ -3,35 +3,42 @@ title: Prettier Code Formatter
 lang: en
 ---
 
-Prettier code formatter supports standard file formats such as JSON, Markdown, HTML, and  JavaScript. Prettier can also support Apex, Aura, and Lightning Web Components (LWC) if you install [Prettier Apex plugin](https://github.com/dangmai/prettier-plugin-apex) contributed by [Dang Mai](https://github.com/dangmai).
+Prettier code formatter supports Aura and Lightning Web Components (LWC) as well as standard file formats such as JSON, Markdown, HTML, and JavaScript. Prettier can also support Apex if you install [Prettier Apex plugin](https://github.com/dangmai/prettier-plugin-apex) authored by [Dang Mai](https://github.com/dangmai).
```

Comment:
Made changes based on @brpowell comments in Gdoc (https://docs.google.com/document/d/1hSxjlzVot7Tf5V0Z89gKbzPdLPQR2CUV0c1HZLFHOMU/edit#)